extensions/2.0/Khronos/KHR_techniques_webgl/schema/material.KHR_technique_webgl.schema.json

```diff
@@ -0,0 +1,27 @@
+{
+    "$schema" : "http://json-schema.org/draft-04/schema",
+    "title" : "KHR_techniques_webgl material extension",
+    "type" : "object",
+    "description" : "The technique to use for a material and any additional uniform values.",
+    "allOf" : [ { "$ref" : "glTFProperty.schema.json" } ],
+    "properties" : {
+        "technique" : {
+            "allOf" : [ { "$ref" : "glTFid.schema.json" } ],
+            "description" : "The index of the technique."
+        },
+        "values" : {
+            "type" : "object",
+            "description" : "Dictionary object of uniform values.",
+            "properties" : {
+            },
+            "additionalProperties" : {
+                "$ref" : "uniform.value.schema.json"
+            },
+            "default" : {},
```

Comment:
I forget if this came up before, but is to safe to just remove `"default" : {}` throughout?